src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -81,6 +81,11 @@
           text: az storage container create -n MyStorageContainer --fail-on-exist
 """
 
+helps['storage account create'] = """
```

Comment:
We should avoid the exact same command requiring different descriptions in two different API versions as this would indicate that the two commands do different things.

> Does the help file break down?

If we do need to do this, we can cross that bridge if we come to it. An obvious solution would be to have conditional logic in the help files but I don't think it's needed yet.